packages/devtools_app/lib/src/app.dart

```diff
@@ -525,6 +528,67 @@ class SettingsDialog extends StatelessWidget {
   }
 }
 
+class _VerboseLoggingSetting extends StatelessWidget {
+  const _VerboseLoggingSetting();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.only(
+        right: 6.0,
+        bottom: 6.0,
+      ),
+      decoration: const BoxDecoration(
+        border: Border.fromBorderSide(
+          BorderSide(width: 0.5, color: Colors.grey),
+        ),
+      ),
```

Comment:
what is this border for? If we need to add a divider, use the Divider widget

Reply:
Ben left a comment about separating the verbose setting from the other settings, so I tried giving it a border.
The padding is so that the right and bottom aren't so squished.
https://github.com/flutter/devtools/pull/5404/files#diff-adc04d3cf2d446d37511bde6ae1ea5a0de49cae5167c03d67d6fc910b42d430e

Reply:
If it the border is no good I can just add the divider yes :)

Reply:
instead of a border, why don't we use a divider between this setting and the other settings? This is more consistent with settings dialogs elsewhere. Use the PaddedDivider widget